packages/localize/src/localize-theia.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+"use strict";
+const { exec } = require('child_process');
+const fs = require('fs');
+
+/*
+* Command to create the i18n folder for globalization and the localise file to use in theia.
+* i.e.:  localize-theia.ts <package>
+  ex:  localize-theia.ts core
+*/
+function createThis() {
+    if (process.argv.length > 2) {
+        // Read the first additional argument passed to the program
+        const myPackage = process.argv[2];
+        const localizeFile = "localize.ts";
+        const browserPath = "./src/browser/";
+        const i18nPath = "./i18n";
+
+        const i18nExampleFile = i18nPath + "/en.json"
+
+        let packagePath = "";
+        let content = copyRight(); // insert the file copyright to start
+
+        // Set the content of the file
+        packagePath = browserPath;
+        content = content + "\n" + importBrowser() + "\n" + fileContent();
+
+        // Create the file to maintain the globalize message
+        const globalizeFile = packagePath + localizeFile;
+
+        // Create the folder to put the localize file
+        const packageCmd = "shx mkdir -p " + packagePath;
+        exec(packageCmd, (err: Error, stdout: String, stderr: String) => {
+            if (err) {
+                console.log("Error creating the folder: " + packagePath);
+                return;
+            }
+            fs.writeFileSync(globalizeFile, content);
+        });
+
+        // Command to create the i18n folder and store an example file if it does not exist yet
+        const i18nCmd = "shx mkdir -p " + i18nPath;
+        exec(i18nCmd, (err: Error, stdout: String, stderr: String) => {
+            if (err) {
+                console.log("Error creating the i18n folder");
+                return;
+            }
+            if (!fs.existsSync(i18nExampleFile)) {
+                fs.writeFileSync(i18nExampleFile, i18nExample());
+            }
+
+        });
+        console.log("Localize for ( " + myPackage + " ) created.");
+    } else {
+        console.log("ERROR: Please pass on a package");
+    }
+}
+
+// Have the main content of the file
+function fileContent() {
+    return `
+    function localize(prefix: string, selection: string, defaultSelection: string): string {
+        return MessageTranslation.localize(prefix + "/" + selection, defaultSelection);
```

Comment:
we should fetch the MessageTranslation instance though a global exported var here.